src/pages/Donate/DonateFormDropInPaymentWrapper.vue

```diff
@@ -231,28 +231,23 @@ export default {
 @import "settings";
 
 .donate-form-dropin-payment-wrapper {
-	.dropin-submit {
-		width: 100%;
-		margin-top: 1rem;
-
-		.icon-lock {
-			height: rem-calc(20);
-			width: rem-calc(20);
-			fill: white;
-			top: rem-calc(3);
-			position: relative;
-			margin-right: rem-calc(8);
-		}
+	.icon-lock {
```

Comment:
Do we need these loading and icon styles anymore?

Reply:
Good call, these are handled in the global drop in style, fixed the wrapper class to get these to display